src/views/WebsitePage/index.js

```diff
@@ -32,13 +32,27 @@ export default class WebsitePage extends Component {
         />
 
         <BlueBackground title={true}>
-          <h1>The</h1>
-          <h1>Golden</h1>
-          <h1>Hack 2.0</h1>
-          {/* <h4>The hackathon for entrepreneurs.</h4> */}
-          <h4>October 3 - 4, 2020.</h4>
-          <h4>Wilfrid Laurier University.</h4>
-          <h4>Now entirely online.</h4>
+          {/* Idk how to have multiple classNames so here's another little inline style for ya */}
```

Comment:
This might be of interest 
[Multiple classNames](https://stackoverflow.com/questions/34521797/how-to-add-multiple-classes-to-a-reactjs-component
)